kratos/python/add_mesh_to_python.cpp

```diff
@@ -286,31 +286,35 @@ template< class TObject >
 pybind11::list GetValuesOnIntegrationPointsArray1d( TObject& dummy,
         const Variable<array_1d<double,3> >& rVariable, const ProcessInfo& rCurrentProcessInfo )
 {
-    pybind11::list values_list;
-    IntegrationPointsArrayType integration_points = dummy.GetGeometry().IntegrationPoints(
-                dummy.GetIntegrationMethod() );
-    std::vector<array_1d<double,3> > values( integration_points.size() );
-    dummy.CalculateOnIntegrationPoints( rVariable, values, rCurrentProcessInfo );
-    for( unsigned int i=0; i<values.size(); i++ )
+    std::vector<array_1d<double, 3>> Output;
```

Comment:
This function is in the core. Is this okay for the other applications? @philbucher

Reply:
This is a separate PR. #6499 .